src/components/common/CategoriesDropdown/index.tsx

```diff
@@ -1,84 +1,148 @@
-import React from 'react'
+import { useQuery } from '@apollo/react-hooks'
+import lodashOrderby from 'lodash.orderby'
+import React, { useEffect, useMemo, useState } from 'react'
 
 import { ButtonSelect } from 'components/buttons/ButtonSelect'
 import { Dropdown, DropdownPosition } from 'components/common/Dropdown'
 import { SelectItem } from 'components/form/SelectItem'
+import { queryTopCategories } from 'queries/OMENCategories'
+import { GetCategories } from 'types/generatedGQLForOMEN'
+import { Remote } from 'util/remoteData'
+import { capitalizeOnlyFirstLetter } from 'util/tools'
 import { Categories } from 'util/types'
 
 interface Props {
-  onClick: (value: Categories) => void
-  value: Categories
+  onClick: (value: string) => void
+  value: string
+}
+
+interface CategoryItem {
+  text: string
+  onClick: () => void
+  value: string
 }
 
 export const CategoriesDropdown = ({ onClick, value }: Props) => {
-  const categoryItems = [
-    {
-      text: Categories.businessAndFinance,
-      onClick: () => {
-        onClick(Categories.businessAndFinance)
+  const [categories, setCategories] = useState<Remote<CategoryItem[]>>(Remote.notAsked())
+
+  const categoryItems: CategoryItem[] = useMemo(
+    () => [
+      {
+        text: Categories.businessAndFinance,
+        onClick: () => {
+          onClick(Categories.businessAndFinance)
+        },
+        value: Categories.businessAndFinance,
```

Comment:
Mapping over an enum is not straightforward, but there is another reason to build each item per separate. Maybe if we know we aren't going to add more categories, it is ok!

Reply:
The categories are default, used by Realitio, must not be change it